ic-os/components/BUILD.bazel

```diff
@@ -32,7 +32,7 @@ genrule(
     name = "commit_timestamp_txt",
     # Depend on .git/index so that the commit timestamp is regenerated whenever
```

Comment:
outdated

Reply:
@frankdavid the `.git/index` part is still wrong right? or did I miss something?

Reply:
Right, I totally missed that and thought you meant that the branch was out of date. Fixed.